src/Pages/Main/Main.js

```diff
@@ -2,8 +2,51 @@ import React from "react";
 import "./Main.scss";
 
 class Main extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      mainData: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/commonData.json")
+      .then((res) => {
+        return res.json();
+      })
+      .then((result) => {
+        this.setState({
+          mainData: result.data,
+        });
+      });
+  }
```

Comment:
```suggestion
      .then((res) => res.json())
      .then((result) => this.setState({
          mainData: result.data,
        }));
  }
```
애로우 펑션이 리턴하는 값이 한줄이라면 축약형으로 써줄 수 있겠네요 :)